.gitignore

```diff
@@ -30,4 +30,4 @@
 echo coverage >> .gitignore
```

Comment:
Can you remove this line in your .gitignore? I accidentally added it to everyones because I thought that was how one adds the coverage to the git ignore file, but it does not work. I have removed it from my file and just want to make sure it does not get merged again.

Reply:
Will not remove for this pull request due to difficulties removing locally and trying to push branch back up. Can remove once we have all pulled the branch down locally.